src/types.ts

```diff
@@ -357,6 +357,9 @@ export interface PoolParametersV1 {
 }
 
 export interface TimeParametersV1 {
+    /**
+     * In epochs
+     */
     rewardPeriodLength: Epoch;
```

Comment:
:thinking:

Reply:
Haha, the type aliases are removed in the outputted types, so this serves as documentation in it's place.